oncotator/configs/column_collapse.config

```diff
@@ -0,0 +1,31 @@
+# This file is a config file for ColumnCollapser instances.
+#
+#  key:value is <name of the annotation to collapse>:<method of collapse>
+#
+#  Currently, the method of collapse can only be: "min" or "mean" (without quotes).  If left blank, mean will be selected.
+#
+[columns_to_collapse]
+ALT_F2R1: min
+i_ALT_F2R1: min
+REF_F2R1: min
+i_REF_F2R1: min
+ALT_F1R2: min
+i_ALT_F1R2: min
+REF_F1R2: min
+i_REF_F1R2: min
+t_Foxog: mean
+i_t_Foxog: mean
+i_tumor_f: mean
+tumor_f: mean
+t_ref_count: min
+t_alt_count: min
+i_t_ref_count: min
+i_t_alt_count: min
+n_ref_count: min
+n_alt_count: min
+i_n_ref_count: min
+i_n_alt_count: min
+
+# These columns are for testing
+hamilcar: mean
```

Comment:
Why are these here?  Shouldn't they go in a test config instead?  If they are here, maybe make them more unlikely somehow?  barca seems close enough to brca that it could somehow sneak into a real file.